en/docs/reference/connectors/smpp-connector/smpp-connector-example.md

```diff
@@ -19,36 +19,31 @@ The following diagram shows the overall solution. There is an HTTP API that you
 
 If you do not want to configure this yourself, you can simply [get the project](#get-the-project) and run it.
 
-## Configure the connector in WSO2 Integration Studio
+## Setup the Integration Project
 
-Connectors can be added to integration flows in [WSO2 Integration Studio](https://wso2.com/integration/integration-studio/). Once added, the operations of the connector can be dragged onto your canvas and added to your resources.
+Follow the steps in the [Creating an Integration Project]({{base_path}}/develop/create-integration-project/) guide to set up the Integration Project. 
 
-### Import the connector
+## Add integration logic
 
-Follow these steps to set up the ESB Solution Project and the Connector Exporter Project. 
+Create an integration project. Then, follow the [Creating a REST API]({{base_path}}/develop/creating-artifacts/creating-an-api/) guide to create a REST API. Specify the API name as `SmppTestApi` and API context as `/smpptest`.
 
-{!includes/reference/connectors/importing-connector-to-integration-studio.md!} 
+### Configuring the API
 
-### Add integration logic
+1. Delete the default resource. <br/>
```

Comment:
Let's remove this part. Or else at the end of the tutorial you can mention remove the default resource optionally.